utilities/infra.py

```diff
@@ -446,6 +446,10 @@ def wait_for_consistent_resource_conditions(
     )
     try:
         for sample in samples:
+            if not sample:
+                LOGGER.warning(f"resource_kind={resource_kind.__name__}: Does not exist yet")
+                continue
+
```

Comment:
Maybe it would be better to call `wait()` on the resource before calling the `wait_for_consistent_resource_conditions()`? So if the resource is not created in a given timeout (which probably should be shorter), then we don't even start to wait for the consistent conditions, wdyt?

Reply:
fair point, done.